src/app/pages/news/11.3/03-engine-cockpit.md

```diff
@@ -1,6 +1,10 @@
 ## Engine Cockpit {#engineCockpit}
 
-New SSL View, TLS tester...
+The Cockpit includes new features to ease your secure operations.
+
+- **Binaries**: The Config File Editor now also lists binary key- and trust-store files. Using the new up-/download functionality, you can edit them locally with a tool of your preference and put the modified version back in charge later on.
+- **TLS Tester**: Secure TLS connections from the engine to remote services can now be tested directly in the cockpit. We list supported cyphers and used certificates. You get that feature no only on REST and SOAP Webservices, but for LDAPS connections too.
```

Comment:
cyphers -> ciphers
no only -> not only
WebServices, -> WebServices      (without comma)